mobidam-sst-management-backend/src/main/resources/application-db-postgres.yml

```diff
@@ -0,0 +1,8 @@
+spring:
```

Comment:
Das sehe ich eher in den umgebungsspezifischen yamls. Ausserdem Haben wir in allen Umgebungen postgres, da brauchen wir doch kein extra Profil dafür.